packages/formatter-html/src/assets/js/scan/scanner-common.js

```diff
@@ -300,9 +339,48 @@
         dateElement.textContent = isoDate.getFullYear() + '-' + pad(isoDate.getMonth() + 1) + '-' + pad(isoDate.getDate()) + ' ' + pad(isoDate.getHours()) + ':' + pad(isoDate.getMinutes());
     };
 
+    var contractDetails = function () {
+        var elements = Array.prototype.slice.apply(document.querySelectorAll('details[open]'));
```

Comment:
What's our browser support matrix in this context? These days I'd normally suggest using `Array.from` unless we need to support IE:
```suggestion
        var elements = Array.from(document.querySelectorAll('details[open]'));
```

Reply:
We are supporting IE11.